### PR TITLE
Use suffixes in memory and disk size

### PR DESCRIPTION
**Pull request by machacekondra**
This PR add a suffix to memory and disk attributes, so it is better
readable for the user instead of using bytes.

Signed-off-by: Ondra Machacek <omachace@redhat.com>

### CODE REVIEW

**Comment by machacekondra** [comment]
Fixes: https://github.com/kubevirt/vm-import-operator/issues/72

File: pkg/providers/ovirt/mapper/mapper.go [hunk 1]
```diff
@@ -164,7 +163,7 @@ func (o *OvirtMapper) MapDisks() map[string]cdiv1.DataVolume {
 		diskID, _ := disk.Id()
 		sdClass := o.getStorageClassForDisk(disk, o.mappings)
 		diskSize, _ := disk.ProvisionedSize()
-		quantity, _ := resource.ParseQuantity(strconv.FormatInt(diskSize, 10))
+		quantity, _ := resource.ParseQuantity(utils.FormatBytes(diskSize))
```
Comment (jakub-dzon) [inline pkg/providers/ovirt/mapper/mapper.go]: mapper tests are failing due to that change.

File: pkg/utils/utils.go [hunk 2]
```diff
@@ -115,3 +116,22 @@ func CountImportedDataVolumes(dvsDone map[string]bool) int {
 
 	return done
 }
+
+// FormatBytes convert bytes to highest suffix
+func FormatBytes(bytes int64) string {
+	if bytes < 0 {
+		return ""
```
Comment (jakub-dzon) [inline pkg/utils/utils.go]: What do you think abou returning error instead?

File: pkg/utils/utils.go [hunk 3]
```diff
@@ -115,3 +116,22 @@ func CountImportedDataVolumes(dvsDone map[string]bool) int {
...
+	if bytes < kib {
+		return fmt.Sprintf("%d", bytes)
+	}
+	suffix := 0
+	n := int64(kib)
+	for i := bytes / kib; i >= kib; i /= kib {
+		n *= kib
+		suffix++
+	}
+	return fmt.Sprintf("%d%ci", bytes/n, "KMGTPE"[suffix])
```
Comment (jakub-dzon) [inline pkg/utils/utils.go]: Is it ok to lose precision? It is very visible  with higher numbers i.e. for `1024*1024*1024*1024-1` you get 1023Gi - that is a difference of 1Gi,

**Review by jakub-dzon — CHANGES_REQUESTED** [review]

File: pkg/utils/utils.go (hunk 3 above)
Comment (machacekondra) [reply pkg/utils/utils.go]: Well, right. I think I will round it. I check and in ovirt it's not possible to create VM with memory like `1024*1024*1024*1024-1`. It always rounded to MiB. For storage size it's rounded to KiB.

**Review by jakub-dzon — APPROVED** [review]

**Review by pkliczewski — APPROVED** [review]

**Pull request merged**